### ISSUE TITLE
Collect user feedback on VZA App 1.0

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#2086]
Email or meet with Lewis to review Scheduler workflow with new location model implemented. 

Specific questions: 
- What information is helpful to have visible on the Locations Details page? 
- Hide School Closure fields when type is street segment or intersection
- GIS Notes? Is this something LL should see, sometimes street segments as described by LL don't line up with street segments in GIS.

- [ ] Create issue(s) for any necessary refinements based on feedback

**Comment by SurbhiBakshi** [comment]
- [ ] Hide School Closure fields when type is street segment or intersection

**Comment by SurbhiBakshi** [comment]
- [ ] What information is helpful to have visible on the Locations Details page?
- School District


- [ ] Hide School Closure fields when type is street segment or intersection

- [ ] GIS Notes? Is this something LL should see, sometimes street segments as described by LL don't line up with street segments in GIS.

 - [ ] Create issue(s) for any necessary refinements based on feedback

**Comment by SurbhiBakshi** [comment]
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/ebf9aba7-e6cd-4d03-9f85-896c8fb346ce)

**Comment by SurbhiBakshi** [comment]
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/cc90c91e-039e-4b71-b6a8-74d6a7ce07bb)

**Comment by SurbhiBakshi** [comment]
Action Items  from review meeting with Lewis on 04/16/2020 

- [ ] LL would like the school status to show up in the concatenated location name if possible and not have to look up school details for closures. Think a filter based on year might work to filter out school locations with any closures.
- [ ] would like to see number of assignments per location so he can make decisions based on that information, a chart or table in the app would be helpful. We have something in the reporting section, but that is not filled out at the moment.
- [ ] While doing the GIS work, I had noticed that some locations are overlapping, these locations might not come across in geographical reporting. We decided that I will send him a list of those locations once I am done with GIS work and he can then decide what to do with those overlapping locations.
- [ ] LL sent me a list of nonAISD schools to include in the location list ( #1440)
